package.json

```diff
@@ -11,7 +11,7 @@
     "lint:fix": "eslint --fix \"app/javascript/src/**/*.{js,jsx}\""
   },
   "lint-staged": {
-    "app/**/*.{js,jsx,json}": [
+    "app/javascript/src/**/*.{js,jsx,json}": [
```

Comment:
Why exactly is this changed required?

Reply:
Got it. But in general this change is not needed. All files under `app` ought to be compliant with eslint and prettier.